### PR TITLE
capacity update South America (BR, BO, CL, UY)

### PR DESCRIPTION
**Pull request by KabelWlan**
## Issue
outdated capacities

## Description
update capacities in BR, BO, CL and UY zones with the sources listed in DATA_SOURCES.md

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -572,8 +572,8 @@
       [-49.37304648399993, 4.941122259000181]
     ],
     "capacity": {
-      "hydro": 22251,
-      "solar": 5,
+      "hydro": 22273,
+      "solar": 3,
```
Comment (VIKTORVAV99) [inline config/zones.json:576]: Huh interesting, do you know why solar capacity was reduced?
(Just curious.)

**Review by VIKTORVAV99 — APPROVED** [review]
I haven't checked *all* the values but those I did check looks good so I'll approve and merge this.

File: config/zones.json (hunk 1 above)
Comment (KabelWlan) [reply config/zones.json:576]: I have no idea, I'm sorry...

Comment (VIKTORVAV99) [reply config/zones.json:576]: No worries!

**Pull request merged**